packages/traceability/issuer/contracts/Registry.sol

```diff
@@ -18,98 +19,93 @@ contract Registry is ERC1155, ERC1888 {
 	// Incrementing nonce, used for generating certificate IDs
     uint256 private _latestCertificateId;
 
-	constructor(string memory _uri) public ERC1155(_uri) {}
+	constructor(string memory _uri) ERC1155(_uri) {}
 
 	/// @notice See {IERC1888-issue}.
     /// @dev `_to` cannot be the zero address.
-	function issue(address _to, bytes calldata _validityData, uint256 _topic, uint256 _value, bytes calldata _issuanceData) external override returns (uint256) {
+	function issue(address _to, bytes calldata _validityData, uint256 _topic, uint256 _value, bytes calldata _data) external override returns (uint256 id) {
 		require(_to != address(0x0), "Registry::issue: to must be non-zero.");
 		
 		_validate(_msgSender(), _validityData);
 
-		uint256 id = ++_latestCertificateId;
-		ERC1155._mint(_to, id, _value, _issuanceData);
+		id = ++_latestCertificateId;
+		ERC1155._mint(_to, id, _value, new bytes(0)); // Check **
```

Comment:
So it's ERC-1888 but using ERC1155._mint method here?

Reply:
ERC-1888 interface is an extension of the ERC-1155 interface. `Registry.sol` is thus and implementation of both ERC-1888 and ERC-1155 and conforms to both.

https://github.com/energywebfoundation/origin/pull/2562/files#diff-a6bfa9bfdd88b1e1a038fde42c571164a468732d40408f39d4aa5caa03d49405R6